CRPPL.g4

```diff
@@ -18,7 +18,7 @@ validexpr: generalquery
 generalquery: RESERVEDWORD (OPERATOR RESERVEDWORD)? LITERAL (SEPARATOR LITERAL)* (RESERVEDWORD LITERAL OPERATOR LITERAL (SEPARATOR LITERAL OPERATOR LITERAL)*)?;
 importfile: RESERVEDWORD RESERVEDWORD SEPARATOR LITERAL SEPARATOR LITERAL SEPARATOR;
 altercolumn: RESERVEDWORD LITERAL RESERVEDWORD LITERAL;
-reportvisualization: DO GRAPH OPENPARENTHESIS LITERAL SEPARATOR LITERAL RESERVEDWORD RESERVEDWORD SEPARATOR LITERAL RESERVEDWORD RESERVEDWORD CLOSEPARENTHESIS;
+graphquery: DO GRAPH OPENPARENTHESIS (TYPE)? SEPARATOR TABLECOLUMN IS LABELONE SEPARATOR TABLECOLUMN IS LABELTWO CLOSEPARENTHESIS;
```

Comment:
TABLECOLUMN can be a literal, but if it's restricted to lowercases, uppercases and underscore, enclose the definition of TABLECOLUMN within a single quote (')